ACE/bin/PerlACE/Process_Unix.pm

```diff
@@ -573,16 +573,17 @@ sub Spawn ()
     return 0;
 }
 
-sub WaitKill ($)
+sub WaitKill ($;$)
 {
     my $self = shift;
     my $timeout = shift;
+    my $opt_signum = shift;
```

Comment:
This should probably be a hashref like I showed [here](https://github.com/objectcomputing/OpenDDS/pull/3416#discussion_r832467568). That would allow us to add more arguments to function without forcing a caller to be concerned with a signal ref argument if they don't need that info.